base/remote/tests/testthat/test.remote.execute.cmd.R

```diff
@@ -0,0 +1,16 @@
+context("Testing remote.execute.cmd() for different possible inputs")
+library(PEcAn.remote)
+library(testthat)
```

Comment:
The usual pattern is to load these packages in `tests/testthat.R`, and not in the individual test files.